tests/Detector.Tests/DotNetCore/SampleProjectFileContents.cs

```diff
@@ -35,5 +35,13 @@ static class SampleProjectFileContents
             <RootNamespace>Microsoft.Oryx.BuildScriptGenerator.Tests</RootNamespace>
           </PropertyGroup>
         </Project>";
+
+        public const string ProjectFileWithOutputTypePlaceHolder = @"
```

Comment:
add few more cases like
ProjectFileWithoutOutputtypePlaceHolder
ProjectFileWithNullOutputtypePlaceHolder
ProjectFileWithEmptyOutputtypePlaceHolder

Reply:
Do we think the checks in the unit tests would suffice? 
https://github.com/microsoft/Oryx/pull/1058/files/6ab2df967ffdcfe5aadda086254b06330b839c6f#diff-3c310ce937af62ff2c475327a2918ff29832e18bb3588baaabe08ed8a425042dR85 

It checks for null & invalid input. If there's something I might be missing, I'm happy to add them, just wanted to check with you first

Reply:
Added new unit test to detect when `OutputType ` is not present per discussion, the other 2 are already accounted for